index.js

```diff
@@ -43,7 +58,19 @@ function updateSpecFromPackage() {
 
 }
 
-function init(aApiDocsPath) {
+/**
+ * @description serve the openAPI docs with swagger at a specified path / url
+ *
+ * @param {object} options
+ * @param {string} [options.path=api-docs] where to serve the openAPI docs. Defaults to `api-docs`
+ * @param {*} [options.predefinedSpec=undefined]
```

Comment:
What would be the type definition for `predefinedSpec`?

Is it a complex structure? Should we make it to be an object then?

Because I couldn't find a way to enable Express's type definitions for the `app` variable, at least with jsdoc comments. Any plans for typescript btw?:D

Reply:
`predefinedSpec` it is either Swagger specification or a function with one argument producing it. See it's usage in function `patchSpec` line 118.

Reply:
didn't really thought about rewriting to TS

Reply:
Alright, I'll make a note.

Yeah, re-writting might be a bit tedious. I guess I can take a look if I find some extra time sooner or later.